pegasus/migrations/096_drop_unique_constraint_from_forms_source_id_index.rb

```diff
@@ -0,0 +1,22 @@
+Sequel.migration do
+  up do
+    # For some reason this unique index only exists on production, but not staging or test.
+    # Conditionally drop it if it exists.
+    if self.indexes(:forms)[:unique_forms_source_id]
+      alter_table(:forms) do
+        drop_index :source_id, name: :unique_forms_source_id
+      end
+    end
+
+    # Replace with a unique index on source_id and kind (rather than just source_id).
+    alter_table(:forms) do
+      add_index [:source_id, :kind], unique: true
+    end
+  end
```

Comment:
There are only 895 rows with a non-null `source_id` in production. The entire table has 5618447 rows. I was assuming it would be some factor of the first number, but perhaps not? We could split it up. Creating the new index is less important, but removing the blocking one on production is.